tile-join.cpp

```diff
@@ -1061,7 +1062,7 @@ void decode(struct tileset_reader *readers, std::map<std::string, layermap_entry
 		// Then this tile is done and we can safely run the output queue.
 
 		if (readers == NULL || readers->zoom != current.first.z || readers->x != current.first.x || readers->y != current.first.y) {
-			if (tasks.size() > 10 * CPUS) {
+			if (tasks.size() > 100 * CPUS) {
```

Comment:
The change from 100 to 10 in the last PR was accidental, left over from testing whether that would reduce memory usage. Restoring it to 100 does not make a big difference to total running time but does reduce time spent on I/O.